pkg/terraform/exec/plugins/vsphereprivate/resource_vsphereprivate_import_ova.go

```diff
@@ -190,9 +190,6 @@ func findImportOvaParams(client *vim25.Client, datacenter, cluster, datastore, n
 		foundDatastore := false
 		foundNetwork := false
 		hostObj.Properties(ctx, hostObj.Reference(), []string{"network", "datastore", "runtime"}, &hostSystemManagedObject)
-		if err != nil {
```

Comment:
Rather than removing the check for an error, we should be looking at the error returned from the call to `hostObj.Properties` instead of ignoring it.